documentation/blog/2023-09-20-runes.md

```diff
@@ -143,6 +150,8 @@ export function createCounter() {
 </button>
 ```
 
+> NOTE: Runes can only be used in `.svelte.js` or `.svelte.ts` files.
```

Comment:
<img width="125" alt="image" src="https://github.com/sveltejs/svelte/assets/1162160/f1c1dc78-e0a4-4374-abd1-6a8127353769">

```suggestion
> Runes can only be used in `.svelte.js` or `.svelte.ts` files.
```

Reply:
```suggestion
> Runes can only be used in `.svelte` and `.svelte.js`/`.svelte.ts` files.
```

Alternative suggestion - I think it might make sense to group JS and TS together like this.